packages/runner/lib/server.ts

```diff
@@ -5,6 +5,7 @@ import type { Request, Response, NextFunction } from 'express';
 import timeout from 'connect-timeout';
 import type { NangoProps, RunnerOutput } from '@nangohq/shared';
 import { getLogger } from '@nangohq/utils/dist/logger.js';
+import * as Usage from './usage.js';
```

Comment:
Any reason why this is capitalized? I think usage is a quite generic name and wouldn’t give a clue to what it is doing at first glance